### PR TITLE
Users/abmodi/master

### PR DESCRIPTION
**Pull request by abhinitmodi**
Tags for perf testing

### CODE REVIEW

File: Tasks/DeployAzureResourceGroup/AzureResourceManagerHelper.ps1 [hunk 1]
```diff
@@ -28,13 +27,14 @@ function Create-AzureResourceGroup
             $azureCommandArguments = "-Name `"$resourceGroupName`" -ResourceGroupName `"$resourceGroupName`" -TemplateFile `"$csmFile`" -TemplateParameterObject `$csmParametersObject $overrideParameters -Verbose -ErrorAction silentlycontinue -ErrorVariable deploymentError"
             $finalCommand = "`$azureResourceGroupDeployment = $azureCommand $azureCommandArguments"
             Write-Host "$finalCommand"
+            Write-Verbose -Verbose "[Azure Resource Manager]Creating resource group deployment with name $resourceGroupName"
```
Comment (pankajkhanzode) [inline Tasks/DeployAzureResourceGroup/AzureResourceManagerHelper.ps1]: Just a nit pick - can you add a space after ] for readability?

**Comment by pankajkhanzode** [comment]
Signed off.

**Comment by pankagar13** [comment]
Signed off

**Comment by pankagar13** [comment]
merging on behalf of Abhinit modi

**Pull request merged**